js/axis.js

```diff
@@ -0,0 +1,99 @@
+function Axis(canvasId, canvasHeight, canvasWidth, valOfY) {
+
+    this.axisCalculation = function () {
+        var canvas = document.getElementById('myCanvas'),
+            ctx = canvas.getContext('2d');
+        this.canvasWidth = canvasWidth;
+        this.canvasHeight = canvasHeight;
+        var arrOfValue = [],
+            graphHeight = (this.canvasHeight * 97) / 100,
+            origDist = (this.canvasWidth * 10) / 100;
+
+        //craeting the X axis (0,500)
+        ctx.beginPath();
```

Comment:
move canvas related logic to a separate class. For drawing a line create Line.js -> function Line(x1,y1,x2,y2)